src/OpenTK.Compute/OpenCL/CL.cs

```diff
@@ -18,23 +18,27 @@ static CL()
 
 		#region Platform API
 
-		/// https://www.khronos.org/registry/OpenCL/sdk/2.2/docs/man/html/clGetPlatformIDs.html
-
 		/// <summary>
 		/// Introduced in OpenCL 1.0
+		/// See more: https://www.khronos.org/registry/OpenCL/sdk/2.2/docs/man/html/clGetPlatformIDs.html
 		/// </summary>
-		/// <param name="numberOfEntries"></param>
-		/// <param name="platforms"></param>
-		/// <param name="numberOfPlatforms"></param>
-		/// <returns></returns>
-
+		/// <param name="numberOfEntries">number of <see cref="CLPlatform"/> entries that can be added to platforms. If platforms is not NULL, numberOfEntries must be greater than zero.</param>
```

Comment:
Usually these start with a capital letter, e.g.
`The number of <see cref="CLPlatform"/> entries ...`